lib/moesif_rack/moesif_middleware.rb

```diff
@@ -144,10 +147,12 @@ def parse_body(body, headers)
 
     def start_worker
       Thread::new do
-        @last_worker_run = Time.now.utc
         loop do
+          # Update the last worker run
+          @last_worker_run = Time.now.utc
```

Comment:
This should be part of inner loop for each batch. As long as thread running, we should update last run time.

Reply:
@dgilling If the @last_worker_run updated for each batch. We will also created another thread when we reach 60 second if the thread is running. Is that what we expect?

Reply:
I think it should be at both places:
1. In queue is empty - the set at the top outside batch loop take care of it.
2. When queue is non-empty - the inner loop will keep updating the runtime.

Reply:
Yes that is correct

Reply:
Added